datacenterlight/migrations/0007_stripeplan.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2017-08-16 19:47
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('datacenterlight', '0006_vmtemplate'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StripePlan',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('stripe_plan_id', models.CharField(max_length=100, null=True)),
```

Comment:
I suggest to use a longer field length, as the plans might become a bit longer over time. I suggest 256 characters